scripts/infra/instructlab-cloud-config-example

```diff
@@ -0,0 +1,18 @@
+EC2_INSTANCE_TYPE="m4.large"
```

Comment:
The specs look relatively small depending on what we want to do. Can we add a section to guide users on choosing the optimal cloud instance type for their usage? It looks m4.large does not have any GPU, we could recommend a type that can take advantage of that?

Reply:
I added additional instance type recommendations for EC2 and IBM